extensions/focustask/src/api/helpers.ts

```diff
@@ -0,0 +1,69 @@
+import {getPreferenceValues} from "@raycast/api"
+import fetch from "node-fetch"
+import {
+  ChecklistResponse,
+  CreateTaskResponse,
+  TasksResponse,
+  TaskStatus,
+} from "./types"
+import {pickBy} from "lodash"
+
+export const getApiRoot = () => {
+  return "http://localhost:3000"
```

Comment:
Is this correct?

Reply:
I'm sorry - I didn't realize I was committing to this repo. WIP - I'll re-request review once done! Thanks!